src/coreclr/vm/exceptionhandling.cpp

```diff
@@ -5191,11 +5191,11 @@ BOOL HandleHardwareException(PAL_SEHException* ex)
         Thread *pThread = GetThread();
         if (pThread != NULL && g_pDebugInterface != NULL)
         {
-            // On ARM and ARM64 Linux exception point to the break instruction.
+#if (defined(TARGET_ARM) || defined(TARGET_ARM64))
```

Comment:
This is in a `TARGET_UNIX` `#ifdef`. It needs to be enabled on MacOS & Linux.  I am not aware of another Unix platform where it needs to be disabled.